web/src/containers/activity/StandardsAndConditions.js

```diff
@@ -1,62 +1,292 @@
 import PropTypes from 'prop-types';
-import React, { Component } from 'react';
+import React from 'react';
 import { connect } from 'react-redux';
 
-import { updateActivity as updateActivityAction } from '../../actions/activities';
+import {
+  setActivityStandardsBusinessResults,
+  setActivityStandardsDocumentation,
+  setActivityStandardsIndustryStandards,
+  setActivityStandardsInteroperability,
+  setActivityStandardsKeyPersonnel,
+  setActivityStandardsLeverage,
+  setActivityStandardsMinimizeCost,
+  setActivityStandardsMITA,
+  setActivityStandardsMitigationStrategy,
+  setActivityStandardsModularity,
+  setActivityStandardsReporting
+} from '../../actions/editActivity';
+
 import RichText from '../../components/RichText';
 import Instruction from '../../components/Instruction';
 import { Subsection } from '../../components/Section';
-import { selectActivityByKey } from '../../reducers/activities.selectors';
-import { STANDARDS } from '../../util';
-
-class StandardsAndConditions extends Component {
-  sync = name => html => {
-    const { activity, updateActivity } = this.props;
-    const updates = { standardsAndConditions: { [name]: html } };
-    updateActivity(activity.key, updates);
-  };
-
-  render() {
-    const { activity } = this.props;
-
-    return (
-      <Subsection resource="activities.standardsAndConditions" nested>
-        {STANDARDS.map(std => (
-          <div key={std.id} className="ds-u-margin-bottom--6">
-            <Instruction
-              source={`activities.standardsAndConditions.${std.id}.instruction`}
-              headingDisplay={{
-                level: 'h6',
-                className: 'ds-h5'
-              }}
-            />
-
-            <RichText
-              content={activity.standardsAndConditions[std.id]}
-              onSync={this.sync(std.id)}
-              editorClassName="rte-textarea-l"
-            />
-          </div>
-        ))}
-      </Subsection>
-    );
-  }
-}
+import { selectActivityByIndex } from '../../reducers/activities.selectors';
+
+const StandardsAndConditions = ({
+  activity,
+  activityIndex,
+  setBusinessResults,
+  setDocumentation,
+  setIndustryStandards,
+  setInteroperability,
+  setKeyPersonnel,
+  setLeverage,
+  setMinimizeCost,
+  setMITA,
+  setMitigationStrategy,
+  setModularity,
+  setReporting
+}) => {
+  const syncBusinessResults = html => {
```

Comment:
In other places, we've used something like:

```js
const getHandleConditions = action => html => action(activityIndex, html)
```

Can we use that here to cut down on the code?